### ISSUE TITLE
Feature Request: Import Content Blocklist

### ISSUE THREAD

**Issue by oakbrad** [Jafner/5etools-docker#19]
Similar to importing homebrew content to patch the site, it would be amazing if we could also import a content block list automatically. The content block list tool exports a json file.

Being able to auto load homebrew and a block list would allow instances to be highly customized to a specific setting / game / homebrew world / etc.

**Comment by Jafner** [comment]
I like that idea a lot. Would be super useful to run separate instances for players which pre-filters the content to what they're allowed to see.  

But AFAIK the site doesn't already support that. Best case, that functionality gets implemented upstream. If that never happens, it may be possible to hack it in by patching a file or two.

**Comment by oakbrad** [comment]
I couldn't find any documentation on this either, but when I set up Plutonium recently I noticed it said to add the blocklist to the index.json when importing homebrew. 

I asked on their Discord and according to the devs [this also works on the website](https://discord.com/channels/363680385336606740/531119028249427997/1266464635104792679). 

So it appears this does already exist upstream and should work same as importing homebrew.
